swift/Samples/ModalContainer/Sources/ModalContainerScreen.swift

```diff
@@ -0,0 +1,53 @@
+import WorkflowUI
+
+/// A `ModalContainerScreen` displays a base screen and optionally one or more modals on top of it.
+public struct ModalContainerScreen<BaseScreen: Screen>: Screen
+{
+
+    /// The base screen to show underneath any modally presented screens.
+    public var baseScreen: BaseScreen
```

Comment:
@dhavalshreyas @JustinDSN @bencochran  using generics in ModalContainerScreen as mentioned in the earlier comments in [PR](https://github.com/square/workflow/pull/1087).